src/ua.c

```diff
@@ -1263,6 +1263,8 @@ int ua_state_json_api(struct odict *od, struct re_printf *pf, const struct ua *u
 
 	/* user-agent info */
 	err |= odict_entry_add(od, "cuser", ODICT_STRING, ua->cuser);
+	/* TODO: 'selected_ua' not working, it's always true? */
+	err |= odict_entry_add(od, "selected_ua", ODICT_BOOL, ua == uag_current());
```

Comment:
The comparing of ua with uag_current() seems to always be true, maybe I'm referencing the wrong thing here.